ngraph/python/src/ngraph/opset7/ops.py

```diff
@@ -83,3 +83,45 @@ def roll(
     inputs = as_nodes(data, shift, axes)
 
     return _get_node_factory_opset7().create("Roll", inputs)
+
+
+@nameable_op
+def dft(
+        data: NodeInput,
+        axes: NodeInput,
+        signal_size: Optional[NodeInput] = None,
+) -> Node:
+    """Return a node which performs NonMaxSuppression.
```

Comment:
I belive it returns not NonMaxSuppression

Reply:
Fixed.